api/app/signals/apps/signals/fields.py

```diff
@@ -118,3 +118,17 @@ def get_object(self, view_name, view_args, view_kwargs):
         return self.get_queryset().get(
             main_category__slug=view_kwargs['slug'],
             slug=view_kwargs['sub_slug'])
+
+
+class NoteHyperlinkedRelatedField(serializers.HyperlinkedIdentityField):
```

Comment:
The class is called `...RelatedField` but the inheritance class is called `...IdentityField`